pulp_ansible/app/viewsets.py

```diff
@@ -267,7 +267,7 @@ def sync(self, request, pk):
 
         result = dispatch(
             sync_func,
-            [repository, remote],
+            exclusive_resources=[repository, remote],
```

Comment:
remote can be among shared_resources.

Reply:
I drafted this to be the minimal change to unbreak master, but i can add shared_resources if desired. Not sure if it were a feature though.